module/PowerShellEditorServices/Start-EditorServices.ps1

```diff
@@ -247,16 +250,16 @@ function Set-NamedPipeMode {
         $PipeFile
     )
 
-    if ($IsWindows) {
+    if (($PSVersionTable.PSVersion.Major -le 5) -or $IsWindows) {
         return
     }
 
     chmod $DEFAULT_USER_MODE $PipeFile
 
-    if ($IsLinux) {
+    if (($PSVersionTable.PSVersion.Major -ge 6) -and $IsLinux) {
```

Comment:
These 2 `6` checks seem unnecessary when the function already returns if it's `5` or lower (line 253).

Reply:
Hehe, good point. I was doing search & replace and didn't even notice that version check a few lines above.  Fixed.